### ISSUE TITLE
RawBencher doesn't compile when new clone of repository

### ISSUE THREAD

**Issue by jonnybee** [FransBouma/RawDataAccessBencher#62]
RawBencher.csproj does not compile a new clone of repository as there is checks for packages that no longer exist in packages folder. 

In my new clone of repository and using VS2022 i only have the folders in packages:
````
25.07.2022  11:22    <DIR>          .
25.07.2022  08:42    <DIR>          ..
25.07.2022  08:42    <DIR>          EntityFramework.6.4.4
25.07.2022  08:42    <DIR>          linq2db.4.1.1
25.07.2022  08:42    <DIR>          Microsoft.Bcl.AsyncInterfaces.6.0.0
25.07.2022  08:42    <DIR>          System.Runtime.CompilerServices.Unsafe.6.0.0
25.07.2022  08:42    <DIR>          System.Threading.Tasks.Extensions.4.5.4`
````

but RawBencher.csproj has these checks for folders: 
````
<Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
  <PropertyGroup>
    <ErrorText>This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
  </PropertyGroup>
  <Error Condition="!Exists('..\packages\NETStandard.Library.2.0.3\build\netstandard2.0\NETStandard.Library.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\NETStandard.Library.2.0.3\build\netstandard2.0\NETStandard.Library.targets'))" />
  <Error Condition="!Exists('..\packages\EntityFramework.6.4.4\build\EntityFramework.props')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\EntityFramework.6.4.4\build\EntityFramework.props'))" />
  <Error Condition="!Exists('..\packages\EntityFramework.6.4.4\build\EntityFramework.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\EntityFramework.6.4.4\build\EntityFramework.targets'))" />
  <error condition="!exists('..\packages\microsoft.codeanalysis.analyzers.3.3.0\build\microsoft.codeanalysis.analyzers.props')" text="$([system.string]::format('$(errortext)', '..\packages\microsoft.codeanalysis.analyzers.3.3.0\build\microsoft.codeanalysis.analyzers.props'))" />
  <error condition="!exists('..\packages\microsoft.codeanalysis.analyzers.3.3.0\build\microsoft.codeanalysis.analyzers.targets')" text="$([system.string]::format('$(errortext)', '..\packages\microsoft.codeanalysis.analyzers.3.3.0\build\microsoft.codeanalysis.analyzers.targets'))" />
  <error condition="!exists('..\packages\microsoft.data.sqlclient.sni.2.1.0\build\net46\microsoft.data.sqlclient.sni.targets')" text="$([system.string]::format('$(errortext)', '..\packages\microsoft.data.sqlclient.sni.2.1.0\build\net46\microsoft.data.sqlclient.sni.targets'))" />
</Target>
````

**Comment by FransBouma** [comment]
Ah good catch! I didn't notice the problems but I had an existing folder. 

Will merge tomorrow and address the other issues

**Comment by jonnybee** [comment]
Merged in PR #63
